packages/models-library/src/models_library/utils/json_schema.py

```diff
@@ -0,0 +1,95 @@
+"""
+Models and schemas are intimately related.
+Here we implement an api on top of jsonschema package adapted for validation of
+json-schemas.
+
+SEE service models
+"""
+# TODO: next step on top could be to construct pydantic models from json-schemas using datamodel-code-generator?
+
+from copy import deepcopy
+from typing import Tuple
+
+import jsonschema
+from jsonschema import validators
+
+# ERRORS
+
+# alias of jsonschema.exceptions._Error
+InvalidJsonSchema = jsonschema.SchemaError
+JsonSchemaValidationError = jsonschema.ValidationError
+
+
+# TODO:  create PydanticValueError and can be raised inside validator
+# create also handlers with context managers?
```

Comment:
Thought: these todos are very specific to you. I am a bit concerned that anyone else is unable to start anything with this kind of comments either by lack of time or lack of understanding of the complexity. Shall we define something such as when it is the case there should be a link to an issue or that these should go somewhere else? especially if the timeline as to when this might be done is totally unknown? If you intend to fix it very soon (like this sprint or the next) just forget this comment